README.md

```diff
@@ -1,179 +1,10 @@
-# Captains Log
+# Captain's Log
```

Comment:
I was wondering if/when this would get an apostrophe...

Reply:
Yeah, I was behind the ball on this one.